tests/zfs-tests/tests/functional/rsend/rsend_011_pos.ksh

```diff
@@ -45,9 +45,6 @@ verify_runnable "both"
 
 function cleanup
 {
-	export __ZFS_POOL_RESTRICT="$POOL $POOL2"
-	log_must zfs unmount -a
-	unset __ZFS_POOL_RESTRICT
```

Comment:
Performing that `zfs unmount -a` will unmount `$POOL1`, `$POOL2`, and the datasets within them just to destroy those datasets and remount `$POOL1` and `$POOL2`. This was causing hangs and the unmount and remount doesn't seem necessary.